src/middleware/guard/token.guard.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  Injectable,
+  HttpException,
+  HttpStatus,
+} from '@nestjs/common';
+import { AuthMappingService } from '@app/modules/authMapping/services/authMapping.services';
```

Comment:
can use the path defined for the module authMapping instead of fetching imports always from @app/modules...
e.g.
port { AuthMappingService } from '@auth-mapping/services/authMapping.services';

Reply:
Good catch!

For hierarchal imports in `tsconfig.json`, NodeJS defaults to the parents most path, which in our case is @app. Treating `/src` makes imports easier.

A better approach might be removing `/src` from paths and using direct module names instead. That will create default imports by module names. What do you think?

Reply:
But I believe it still works the way we are using, No?

Reply:
Oh, never mind. I was referring to the default imports starting with `@app/...` However, I realized that if I reorder the `paths` in `tsconfig.json` and move `/src` to the bottom, VS Code will, by default, pick the first matching path it finds, which will correspond to the path defined for `module_name`.

I'll update this in my new PR.